dts/bindings/clock/silabs,series2-lfxo.yaml

```diff
@@ -0,0 +1,29 @@
+compatible: "silabs,series2-lfxo"
+
+description: |
+  Silicon Labs LFXO peripheral.
+
+include: fixed-clock.yaml
+
+properties:
+  ctune:
+    type: int
+    required: true
+    description: Load capacitance configuration
+  precision:
+    type: int
+    required: true
+    description: Precision configuration
+  timeout:
+    type: int
+    required: true
+    description: Startup timeout delay (cycles)
+    enum: [2, 256, 1024, 2048, 4096, 8192, 16384, 32768]
+  mode:
+    type: string
+    description: |
+      HFXO mode. Defaults to "xtal", expecting a 32768 Hz crystal oscillator on XI and XO pins.
```

Comment:
HFXO? (BTW, you can probably get rid of this description. "LFXO mode" is not very helpful when the attribute is named "mode" and the device "LFXO").